java/daikon/Chicory.java

```diff
@@ -130,6 +130,9 @@
   @Option("Number of calls after which sampling will begin")
   public static int sample_start = 0;
 
+  @Option("Enable remote debug")
```

Comment:
Please add a Javadoc comment, explaining what this does or how to use it.

Reply:
This was added and used by Philipp Hirch.  I do not know how to use it.  Should I comment out or just delete?

Reply:
This is not a new change -- it already existed, and he just moved it to be a command-line option.